zerver/lib/cache.py

```diff
@@ -581,11 +585,13 @@ def delete_user_profile_caches(user_profiles: Iterable["UserProfile"]) -> None:
 
     keys = []
     for user_profile in user_profiles:
-        keys.append(user_profile_by_email_cache_key(user_profile.delivery_email))
```

Comment:
Why does `user_profile_by_email_cache_key(user_profile.email)` not appear in this block?

Reply:
Looks like a mistake made when restructuring commits - this line should be deleted in the commit deleting the cache function. Fixed now